app/assets/stylesheets/govuk_publishing_components/components/_document-list.scss

```diff
@@ -32,6 +32,10 @@
   float: left;
   list-style: none;
   padding-right: $gutter-two-thirds;
+
+  .direction-rtl & {
+    float: right;
```

Comment:
Instead of floating this right you could simply take the float left off the original element style and replace it with display inline-block - it's then influenced by the text direction but uses less CSS (of course, I haven't tested this cross browser but I think it should work).

Reply:
That sounds good, I wonder why we used float to start with.. I'll have a go 👍